infoschema/tables.go

```diff
@@ -147,6 +148,7 @@ var tablesCols = []columnInfo{
 	{"CHECKSUM", mysql.TypeLonglong, 21, 0, nil, nil},
 	{"CREATE_OPTIONS", mysql.TypeVarchar, 255, 0, nil, nil},
 	{"TABLE_COMMENT", mysql.TypeVarchar, 2048, 0, nil, nil},
+	{"TIDB_TABLE_ID", mysql.TypeLonglong, 21, 0, nil, nil},
```

Comment:
s/TIDB_TABLE_ID/TABLE_ID?

Reply:
better not, it's special column not standard column